lightning/src/ln/channelmanager.rs

```diff
@@ -9559,13 +9569,19 @@ where
 							None
 						}
 					},
+				};
+
+				match (responder, response) {
+					(Some(responder), Some(response)) => return responder.respond(response),
```

Comment:
Let's log if we have a response but no responder.

Reply:
Introduced a `log_trace!` in [77c393a](https://github.com/lightningdevkit/rust-lightning/commit/77c393a6eec06158cbe5fe33608fbe00e2386135)